docs/docs/index.md

```diff
@@ -1,143 +1,134 @@
-- [Posit Connect](https://www.posit.co/download/posit-connect/) v1.7.0 or higher, configured with Python support
+- [Python >=3.7](https://www.python.org/downloads/)
+- [Jupyter](https://pypi.org/project/jupyter)
+- [Posit Connect](https://www.posit.co/download/posit-connect/**) v1.7.0 or higher, configured with Python support
 
-!!! note
-    If using `conda`, `pip` and `wheel` should already be installed.
+!!! Warning
+    This extension is **NOT** compatible with _JupyterLab_. Only _Jupyter Notebooks_, and associated runtime enviornments, such as _JupyterHub_, are supported.
 
-## Installation
-
-The installation method depends on the Python environment that you are installing the `rsconnect-jupyter` package into.
+!!! Warning
+    In order to publish to _Posit Connect_, a compatible Python environment must in exist on the _Post Connect_ instance. See the _Posit Connect_ documentation on [Python integrations](https://docs.posit.co/rsc/integration/python/) for additional information.
 
-!!! note
-    The `rsconnect-jupyter` package is developed for Jupyter Notebook, specifically. Therefore, the package does not work with the JupyterLab development environment.
+## Installation
 
-This documentation covers three methods:
+--8<-- "snippets/install.md"
 
-- [Installing Jupyter within a virtual environment](#installing-jupyter-within-a-virtual-environment)
-- [Installing `rsconnect-jupyter` to Jupyter running on Posit Workbench](#installing-to-jupyter-running-on-posit-workbench)
-- [Installation in JupyterHub](#installing-in-jupyterhub)
+## Runtime Environment Configuration
 
-Please navigate to the installation section below that is best for your environment.
+### Localhost (Your Computer)
 
-### Installing Jupyter within a virtual environment
+For localhost installation, a Python virtual environment is recommend to isolate runtime dependencies. There are various Python virtual environments avaiable. The following tutorial covers a few of them.
 
-To install and use Jupyter within a virtual environment using
-`virtualenv`, follow the procedures shown below or learn more using the
-[Virtualenv](https://virtualenv.pypa.io/en/latest/) documentation.
+#### Conda
 
-- These commands create and activate a `virtualenv` at `/my/path`:
-  <div class="code-title">Terminal</div>
-  ```bash
-  $ pip install virtualenv
-  virtualenv /my/path
-  source /my/path/bin/activate
-  ```
+Install `rsconnect-jupyter` from [Conda Forge](https://conda-forge.org).
 
-!!! tip
-    Running `source /my/path/bin/activate` activates the virtual environment. While the `virtualenv` is active, Python-related commands like `python`, `pip`, and `jupyter` will use to copies located inside the virtual environment. You can check which copy of `python` you're using by running `which python`.
+<div class="code-title">Terminal</div>
+```shell
+conda create --name rsconnect-jupyter
+conda activate rsconnect-jupyter
+conda install -c conda-forge jupyter rsconnect-jupyter
+```
```

Comment:
Anaconda used to (and may still) install a jupyter extension that creates a kernelspec for every local conda environment that contains a kernel package. When you go to create a new notebook in Jupyter, you can choose from the various kernels you have available. So, the notebook server can be in one environment and the ipykernel can be in another.

We currently require that rsconnect-python be installed in the kernel environment (as well as being in the notebook server environment along with rsconnect-jupyter) so that environment inspection works and finds the correct packages. It's certainly possible to change that but at the moment that is the requirement.

I see that's mentioned below under JupyterHub but I don't think it's JupyterHub-specific.